test/test_ci_containment.rb

```diff
@@ -276,22 +257,19 @@ def test_image_is_pangeaimage
 
     def test_ZZZ_binds # Last test always! Changes VCR configuration.
       # Container binds were overwritten by Containment at some point, make
-      # sure the binds we put in are the binds that are passed to docker.
-      Dir.chdir(@tmpdir) do
-        Docker::Container.prepend(InterceptStartContainer)
-        InterceptStartContainer.intercept_start_container = true
-        vcr_it(__method__, erb: true, tag: :erb_pwd) do
+      # sure the binds we put in a re the binds that are passed to docker.
+      WebMock.disable_net_connect!
+      VCR.turned_off do
```

Comment:
It needs turning off for some reason, because without it :

```
Error: test_ZZZ_binds(CI::ContainmentTest):
  VCR::Errors::UnhandledHTTPRequestError: 
  
  ================================================================================
  An HTTP request has been made that VCR does not know how to handle:
    GET unix:///var/run/docker.sock/v1.24/version
    Body: 
```